### PR TITLE
views

### PR DESCRIPTION
**Pull request by maths29**
In this project, I created the following;

Basic UI created for pages to show all users fetched from database
Basic UI created for pages to show all Posts fetched from database

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,20 @@
...
+        <div class="row">
+            <div class="col-lg-12 border border-dark">
+                <% post.recent_comments.each do |comment| %>
+                    <p class=""><%= comment.author.name %>: <%= comment.body %> </p>
+                <% end %>
+
+            </div>
+        </div>
+    <% end %>
+</section>
```
Comment (Gambit142) [inline app/views/posts/index.html.erb]: - [ ] On the Posts#Index page, there should be a button with the text _**Pagination**_ based on the wireframe provided. Kindly include this button. You can decide to include it as a placeholder or fully implement it. The choice is yours to make but the button should be present on the page. 👍 😎 

image

File: app/views/shared/_user.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    <div class="col-lg-2">
+        <%= image_tag("default.jpg", alt: "Default", style: "width: 70%;") %>
+    </div>
+    <div class="col-lg-10 border border-dark d-flex align-items-center">
+        <div class="user-lists w-100">
+            <h2 class="text-dark"><%= @user.name %></h2>
+            <p class="d-flex justify-content-end">Number Of posts <%= @user.posts_counter %></p>
+        </div>
+    </div>
+</div>
```
Comment (Gambit142) [inline app/views/shared/_user.html.erb:11]: - [ ] Great job creating this partial and using it where necessary. 👏

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @maths29 👋, 

Good job so far!

## Highlights
- Setup linters correctly ✔️ 
- Created **_index and show pages_** for the `User` model ✔️ 
- Created **_index and show pages_** for the `Post` model ✔️ 
- Great use of partials ✔️ 
- Great use of custom methods to display certain features ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by julie-ify — APPROVED** [review]
Hi @maths29,

## Highlights!
- Good code organization and styled views ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**